src/main/java/com/homework/week15/sql/generate/User.java

```diff
@@ -0,0 +1,20 @@
+package com.homework.week15.sql.generate;
+
+@Table(name = "users")
+public class User implements Model {
+
+    @Column(name = "full_name")
+    private String name;
+
+
+    @Column(name = "age")
+    private Integer age;
+
+    private Integer year;
```

Comment:
let's assume that all fields should be annotated with Column annotation. I suggest to remove the year filed for now.